client/src/components/chess-game/ChessGame.js

```diff
@@ -14,6 +14,15 @@ import './ChessGame.css';
  * and while running server or building server it exports function
 */
 const Chess = typeof ChessJs === 'function' ? ChessJs : ChessJs.Chess;
+const blackPlayerBoard = (squares, board) => {
+  const newSquares = Object.assign([], squares);
+  let newBoard = Object.assign([], board.map(x => Object.assign([], x)));
```

Comment:
What is x?

Reply:
Write name if it signifies an entity.

Reply:
x is for rows of the chessboard grid. I have renamed it to row.